src/ProofOfConcepts/KPIUpdates/Components/ConsumerGroups.tsx

```diff
@@ -0,0 +1,43 @@
+import OutlinedQuestionCircleIcon from "@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon";
+import { VoidFunctionComponent } from "react";
+import React from "react";
+import {
+  Bullseye,
+  Card,
+  CardBody,
+  CardFooter,
+  CardTitle,
+  Flex,
```

Comment:
Some of the imports in the .tsx files aren't called e.g. here we could take them out ? (although it still works)

Reply:
Removed the unnecessary imports.